cves/CVE-2016-2213.yml

```diff
@@ -188,8 +204,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    The vulnerability was an index out of bounds error, so running a wide
+    variety of test cases would be able to catch this.
```

Comment:
Just some of my thoughts not necessarily anything for correction, if the file directly impacts the For Loop, is it possible that testing the wide arrange of different vulnerabilities a JPEG file could introduce are not able to be automated.